google-analytics-provider/build.gradle

```diff
@@ -49,7 +49,7 @@ android {
 dependencies {
 	implementation project(path: rootProject.ext.analytics_kit)
 	api "org.jetbrains.kotlin:kotlin-stdlib:$rootProject.ext.kotlinVersion"
-	api "androidx.annotation:annotation:1.2.0"
+	api "androidx.annotation:annotation:$rootProject.ext.annotationsVersion"
 	compileOnly 'com.google.android.gms:play-services-analytics:17.0.0'
 
 	testImplementation 'com.google.android.gms:play-services-analytics:17.0.0'
```

Comment:
There is a newer version of the Google Analytics SDK too, but I think there were some breaking changes and I would have had to conform to different function signatures. Let's update to the latest SDK in a follow-up PR.